src/index.js

```diff
@@ -8,30 +8,51 @@ import {runScripts} from './transformScriptTags';
 function processOptions(options) {
   // Parse preset names
   const presets = (options.presets || []).map(presetName => {
-    if (typeof presetName === 'string') {
-      var preset = availablePresets[presetName];
-      if (!preset) {
-        throw new Error(`Invalid preset specified in Babel options: "${presetName}"`);
+    let preset = null;
+
+    if (presetName instanceof Array && typeof presetName[0] === 'string') {
```

Comment:
Yeah, I avoided `Array.isArray` because it would require a polyfill.

I actually had to google "how to check if array without Array.isArray" :D